tests/ci/profile_handler.go

```diff
@@ -421,7 +421,7 @@ func GenerateClusterCreationArgsByProfile(token string, profile *Profile) (clust
 			//		* The Shared-VPC compatible policie requries installer role
 			//		* The install role (account roles) require Shared-VPC ARN.
 			//  Use hardcode as a temporary solution.
-			shared_vpc_role_arn = fmt.Sprintf("arn:aws:iam::641733028092:role/%s-shared-vpc-role", clusterArgs.ClusterName)
+			shared_vpc_role_arn = fmt.Sprintf("arn:aws:iam::641733028092:role/%s-shared-vpc-role", *clusterArgs.ClusterName)
```

Comment:
Can you please create a jira ticket to track the enhancement of this part rather than hardcode the role arn here?

Reply:
create a JIRA card(OCM-8543) to track this.